tools/template/README.md

```diff
@@ -0,0 +1,43 @@
+# wtstats template
+
+This subfolder contains the code to generate the `wtstats.html.template` file needed by the Python script `wtstats.py`, which is located in the `./tools` directory. 
+
+The template is generated as a node.js "single page" application using several open source frameworks, see Dependencies below. The `wtstats.py` Python script parses stats files, transforms the data and inserts it as JSON into the template. The resulting HTML page can be openend in the browser.
+
+### Build Process
+
+To build the template, follow these steps: 
+
+1. change into the `./tools/template` directory
+2. run `npm install` to install all missing dependencies. 
+3. run `npm run build` to build the template, pack everything into a single HTML file and copy it to its parent folder as `./tools/wtstats.html.template`. 
+
+### Usage
+
+Call the `wtstats.py` Python script with a stats file as argument. The `test` subfolder contains a small example stats file to test: 
+
+```
+python wtstats.py ./test/WiredTigerStat.fixture
+```
+
+The script will create `wtstats.html` in the working directory, which can be viewed with any modern browser (latest versions of Chrome, Safari, FireFox, Internet Explorer supported).
+
+For more information about _wtstats_ usage, check out the [wtstats documentation](https://github.com/wiredtiger/wiredtiger/wiki/WiredTiger-statistics) on the WiredTiger wiki.
+
+### Dependencies
+
```

Comment:
For dependencies, should we list the version of each package that we've tried this with?